changelog/unreleased/4448

```diff
@@ -0,0 +1,10 @@
+Enhancement: Improved accessibility of information and relationships
+
+Headings have been added to the following views: Share, Edit/Create Share Link, Standard Toolbar and Manage Accounts.
+The filename input field and the two switches are now linked to the their labels.
+The 'contentDescription' attributes of the buttons in the edit/create share link have also been changed.
```

Comment:
edit/create must be both in capitals or not.

Reply:
For me yes, these are different views. Open to discussion.

Reply:
I mean, in the line 3 you wrote `Edit/Create Share Link` (everything in capitals). In the line 5 you wrote `edit/create share link` (everything in non-capitals). No discussion, both in capitals or both in non-capitals.

Reply:
still differences:

Edit/Create Share Link
Edit/Create share link

Reply:
Ups... done